app/src/main/java/com/google/droidjump/models/Palm.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.droidjump.models;
+
+import android.content.res.Resources;
+import com.google.droidjump.R;
+
+public class Palm extends Obstacle {
+    public Palm(int x, int y, int pictureID, Resources resources) {
+        super(x, y, R.mipmap.palm, resources);
+        this.y = (y - picture.getHeight());
```

Comment:
use the common method for this (y - picture.getHeight());

Reply:
Done